### ISSUE TITLE
Patch issues

### ISSUE THREAD

**Issue by Sadmasty** [dalaranwow/dalaran-wow#5385]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
![patch required wow](https://user-images.githubusercontent.com/38155038/38453630-2bff09b2-3a59-11e8-8162-4c8ac82caa99.jpg)
i am greeted by this message when i try to log into the game client. i press restart and i am greeted by the picture below.
![failed patch restart wow](https://user-images.githubusercontent.com/38155038/38453619-e1b12c32-3a58-11e8-9cd3-dccbf14f5bf0.jpg)

i have reinstalled the game twice with the same results.
  
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. i
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Make sure that you log in with your account username, and not with your e-mail, and it will work.
